### PR TITLE
CNV-30795: small fix prereqs

### PR DESCRIPTION
**Pull request by sabrinajess**
Version(s): 4.13+

Issue: [CNV-30795](https://issues.redhat.com//browse/CNV-30795)

Link to docs preview: https://sabrinajess.github.io/previews/CNV22648/virt/virtual_machines/virt-automating-windows-sysprep.html#virt-generalizing-windows-sysprep_virt-automating-windows-sysprep

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/virt-generalizing-windows-sysprep.adoc [hunk 1]
```diff
@@ -10,6 +10,10 @@ Generalizing an image allows that image to remove all system-specific configurat
 
 Before generalizing the VM, you must ensure the `sysprep` tool cannot detect an answer file after the unattended Windows installation.
 
+.Prerequisites
+
+* A running Windows virtual machine with the QEMU guest agent installed.
```
Comment (jab-rh) [inline modules/virt-generalizing-windows-sysprep.adoc:15]: Would including a link to _[Installing the QEMU guest agent and VirtIO drivers](https://sabrinajess.github.io/previews/CNV22648/virt/virtual_machines/virt-installing-qemu-guest-agent.html)_ in the assembly make sense? Or would a user know what QEMU guest agent is?

**Review by jab-rh — COMMENTED** [review]
@sabrinajess, minor comment, thanks!

File: modules/virt-generalizing-windows-sysprep.adoc (hunk 1 above)
Comment (sabrinajess) [reply modules/virt-generalizing-windows-sysprep.adoc:15]: Not a bad idea - I'll add

Comment (apinnick) [reply modules/virt-generalizing-windows-sysprep.adoc:15]: That would be problematic in a module! :-)

**Comment by geetikakay** [comment]
lgtm

**Pull request merged**